### PR TITLE
Add subgraph shapes

### PR DESCRIPTION
**Pull request by HertzHan**
## PR type
unittest, debug
## description
add shapes for layernorm, rms norm, softmax and rope subgraphs, while fixing bugs in these unittests

### CODE REVIEW

File: framework/e2e/PaddleLT_new/layerApicase/perf_monitor/manual_subgraph/softmax_fp32_shape_128_12_128_128.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    def forward(self, x, axis=-1):
+        output = paddle.nn.functional.softmax(x, axis=axis)
+        return output
+
+
+def create_tensor_inputs():
+    shape = [128, 12, 128, 128]
+    x = paddle.uniform(shape, dtype="float32", min=-0.5, max=0.5)
+    x.stop_gradient = False
+    inputs = x
```
Comment (Zeref996) [inline framework/e2e/PaddleLT_new/layerApicase/perf_monitor/manual_subgraph/softmax_fp32_shape_128_12_128_128.py]: 需要改成 inputs = (x, )    以保持所有子图输入的形式一致

File: framework/e2e/PaddleLT_new/layerApicase/perf_monitor/manual_subgraph/softmax_fp32_shape_1_1_4096.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    def forward(self, x, axis=-1):
+        output = paddle.nn.functional.softmax(x, axis=axis)
+        return output
+
+
+def create_tensor_inputs():
+    shape = [1, 1, 4096]
+    x = paddle.uniform(shape, dtype="float32", min=-0.5, max=0.5)
+    x.stop_gradient = False
+    inputs = x
```
Comment (Zeref996) [inline framework/e2e/PaddleLT_new/layerApicase/perf_monitor/manual_subgraph/softmax_fp32_shape_1_1_4096.py]: 需要改成 inputs = (x, ) 以保持所有子图输入的形式一致

File: framework/e2e/PaddleLT_new/layerApicase/perf_monitor/manual_subgraph/softmax_fp32_shape_128_128_768.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    def forward(self, x, axis=-1):
+        output = paddle.nn.functional.softmax(x, axis=axis)
+        return output
+
+
+def create_tensor_inputs():
+    shape = [128, 128, 768]
+    x = paddle.uniform(shape, dtype="float32", min=-0.5, max=0.5)
+    x.stop_gradient = False
+    inputs = x
```
Comment (Zeref996) [inline framework/e2e/PaddleLT_new/layerApicase/perf_monitor/manual_subgraph/softmax_fp32_shape_128_128_768.py]: 需要改成 inputs = (x, ) 以保持所有子图输入的形式一致

File: framework/e2e/PaddleLT_new/layercase/perf_monitor/manual_subgraph/softmax_fp32_shape_1_1_4096.py [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    def forward(self, x, axis=-1):
+        output = paddle.nn.functional.softmax(x, axis=axis)
+        return output
+
+
+def create_tensor_inputs():
+    shape = [1, 1, 4096]
+    x = paddle.uniform(shape, dtype="float32", min=-0.5, max=0.5)
+    x.stop_gradient = False
+    inputs = x
```
Comment (Zeref996) [inline framework/e2e/PaddleLT_new/layercase/perf_monitor/manual_subgraph/softmax_fp32_shape_1_1_4096.py]: 需要改成 inputs = (x, ) 以保持所有子图输入的形式一致

File: framework/e2e/PaddleLT_new/layercase/perf_monitor/manual_subgraph/softmax_fp32_shape_128_12_128_128.py [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+    def forward(self, x, axis=-1):
+        output = paddle.nn.functional.softmax(x, axis=axis)
+        return output
+
+
+def create_tensor_inputs():
+    shape = [128, 12, 128, 128]
+    x = paddle.uniform(shape, dtype="float32", min=-0.5, max=0.5)
+    x.stop_gradient = False
+    inputs = x
```
Comment (Zeref996) [inline framework/e2e/PaddleLT_new/layercase/perf_monitor/manual_subgraph/softmax_fp32_shape_128_12_128_128.py]: 需要改成 inputs = (x, ) 以保持所有子图输入的形式一致

File: framework/e2e/PaddleLT_new/layercase/perf_monitor/manual_subgraph/softmax_fp32_shape_128_128_768.py [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+    def forward(self, x, axis=-1):
+        output = paddle.nn.functional.softmax(x, axis=axis)
+        return output
+
+
+def create_tensor_inputs():
+    shape = [128, 128, 768]
+    x = paddle.uniform(shape, dtype="float32", min=-0.5, max=0.5)
+    x.stop_gradient = False
+    inputs = x
```
Comment (Zeref996) [inline framework/e2e/PaddleLT_new/layercase/perf_monitor/manual_subgraph/softmax_fp32_shape_128_128_768.py]: 需要改成 inputs = (x, ) 以保持所有子图输入的形式一致

**Review by Zeref996 — APPROVED** [review]
